Figure_05_08.py

```diff
@@ -1,15 +1,38 @@
 import matplotlib.pyplot as plt
 import utilsplot
+import numpy as np
+from scipy.signal import lti, lsim
 
-from utils import tf
+from utils import tf, feedback
 
 s = tf([1, 0])
 
 #Figure 5.8
 G = (-s + 1)/(s + 1)
-Kc = [0.1, 0.5, 0.9]
+Kcs = [0.1, 0.5, 0.9]
 K = -1*s/((1 + 0.02*s)*(1 + 0.05*s))
 
 plt.figure('Figure 5.8')
-utilsplot.freq_step_response_plot(G, K, Kc, 0.2, 'S')
+utilsplot.freq_step_response_plot(G, K, Kcs, 0.2, 'S')
+plt.show()
```

Comment:
by default, `plt.show()` will block, so this script will require you to close the first plot window to see the next one. It's probably better only to call `plt.show()` once at the end.